### PR TITLE
1.0.1

### PR DESCRIPTION
**Pull request by psfoley**
1.0.1 is a patch release. It includes the following updates:
- New docker CI tests
- New Pytorch UNet Kvasir tutorial
- Cleanup / fixes to other OpenFL tutorials
- Fixed description for Pypi
- Status/documentation/community badges for README.md

### CODE REVIEW

**Comment by grib0ed0v** [comment]
@psfoley LGTM!

**Review by alexey-gruzdev — APPROVED** [review]

**Comment by alexey-gruzdev** [comment]
@psfoley LGTM!

File: README.md [hunk 1]
```diff
@@ -20,15 +27,15 @@ across the federation is solely the model parameters and their updates.
 Open Federated Learning (OpenFL) is a Python 3 project developed by Intel Labs and 
 Intel Internet of Things Group. 
 
-![Federated Learning](docs/images/diagram_fl.png)
+![Federated Learning](https://raw.githubusercontent.com/intel/openfl/master/docs/images/diagram_fl.png)
```
Comment (tonyreina) [inline README.md:30]: Shouldn't this just point to `docs/images/diagram_fl.png`?

File: README.md [hunk 2]
```diff
@@ -20,15 +27,15 @@ across the federation is solely the model parameters and their updates.
...
 
 ## Requirements
 
 - OS: Tested on Ubuntu Linux 16.04 and 18.04.
 - Python 3.6+ with a Python virtual environment (e.g. [conda](https://docs.conda.io/en/latest/))
 - TensorFlow 2+ or PyTorch 1.6+ (depending on your training requirements). OpenFL is designed to easily support other frameworks as well.
 
-![fx commandline interface](docs/images/fx_help.png)
+![fx commandline interface](https://raw.githubusercontent.com/intel/openfl/master/docs/images/fx_help.png)
```
Comment (tonyreina) [inline README.md:38]: Again. can't this be a relative link?

File: openfl-docker/Dockerfile.base [hunk 3]
```diff
@@ -3,10 +3,15 @@
 
 # If your machine is behind a proxy, make sure you set it up in ~/.docker/config.json
 
-ARG BASE_IMAGE=python:3.6.12-slim-stretch
+ARG BASE_IMAGE=python:3.8
```
Comment (tonyreina) [inline openfl-docker/Dockerfile.base:6]: Nice.

**Review by tonyreina — APPROVED** [review]
Just a question about the links. Otherwise, LGTM.

File: README.md (hunk 1 above)
Comment (alexey-gruzdev) [reply README.md:30]: @tonyreina this was done for correct PyPI package page rendering.

File: README.md (hunk 2 above)
Comment (alexey-gruzdev) [reply README.md:38]: The same reason as above

**Pull request merged**